packages/backend/src/indexers/BlockNumberIndexer.ts

```diff
@@ -47,6 +60,24 @@ export class BlockNumberIndexer extends ChildIndexer {
   }
 
   async update(_fromTimestamp: number, toTimestamp: number): Promise<number> {
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```

Comment:
Double check if this eslint declaration is still required

Reply:
[e0c40fb](https://github.com/l2beat/lz-monitoring/pull/23/commits/e0c40fbfd06d0445a38aa1716709bb917f6778e6)